### ISSUE TITLE
Include move number in white move

### ISSUE THREAD

**Issue by braoult** [mliebelt/PgnViewerJS#416]
We can see some games (inline mode) where the move number is displayed at the end of a line and white move on next line, or, when displaying moves "justified", space is taken between move number and white move, which is not very easy to read.

Maybe it could be possible to have white move defined as something like (for `1.e4`):
```
<move id="boardMoves0" class=" white">
    <move-number data-value="1">1.</move-number>
    <san>e4</san>
</move>
```
instead of (current) :
```
<move-number data-value="1">1. </move-number>
<move id="boardMoves0" class=" white">
    <san>e4</san>
</move>
```
I am unsure of implications, or even if it is possible, however.
Note: I think lichess is going this way, but I did not check other systems...

**Comment by mliebelt** [comment]
I will check which options we have. Of course CSS is able to do that, but I would like to find the least invasive solution, so that no other things are breaking.

**Comment by braoult** [comment]
Yes, I know breaking things is not an option ! This is why it is better to be cautious here, as you say...

Something else: I had a look to some chess magazines: `Europe Echecs`, `Gambisco/Le Joueur d'Échecs`,  and `Echec et Mat` (official magazine of French Federation). All from late 90s, as was not in France after that; I believe all of they may have disappeared, unfortunately.

In these magazines, games were always printed in the equivalent of our `inline` mode, with white move attached to move number, for example : `1.e4` instead of `1. e4`.

Lichess does not, but does the following:
```
<move>
   <index>1.</index>
   e4
</move>
```
Here, `move index` has the property `padding-right: .2em;`, a fixed distance between move number and white move, then.

Note: I will stop using lichess example alone, this is only one example between many where we can find games displayed; and one example is not an argument :-)

**Comment by mliebelt** [comment]
Do you have an example to reproduce that? I will give it a try, play with different sizes, to see that case happening.

**Comment by braoult** [comment]
> Do you have an example to reproduce that? I will give it a try, play with different sizes, to see that case happening.

I believe any (long) PGN would show the issue. For example [this one](https://raoult.com/chess/showgame.phtml/pgn/Brittany-Open-1984.pgn/2), which shows:

![image](https://github.com/mliebelt/PgnViewerJS/assets/8958429/7470f755-0df1-4e09-8814-837db4a17774)

**Comment by mliebelt** [comment]
I added myself an example where you can see that as well. I have even seen linebreak in "O-O". So I think I will address that as well then. And yes, seems to be a problem with the flat structure of the moves/numbers.
